### ISSUE TITLE
Recognize the difference between a workflow and a simulation

### ISSUE THREAD

**Issue by mostaphaRoudsari** [ladybug-tools/queenbee#39]
This is related to #32 and how hydrating workflows is currently done. Some notes about the terminologies:

## Workflow
A workflow is reusable. It means you can use the workflow over and over by just changing the arguments and the logic stays the same. Workflow is useful to define the process.

## Simulation (for the lack of a better term)
A simulation is a hydrated/frozen version of a workflow. It has assigned values for each input argument which is passed to the workflow in the time that the simulation is created. Also there is no referenced values in a simulation. All the referenced values has been replaced by input values. Simulation is useful to execute the process.

I think we should recognize and respect this difference or otherwise we have two very similar object which will be hard to recognize from each other.

With this logic one should not be able to hydrate a workflow. You can get a hydrated version of the workflow at any point but that should be called something else like `simulation`.

@AntoineDao, I mainly opened this issue to get your feedback on this:

1. Do you agree with this distinction?
2. If yes, what would you call the second one? Should it be a new schema of its own that for example doesn't have assignable inputs?

**Comment by mostaphaRoudsari** [comment]
We can also rename what we call `workflow` now to `recipe` and call a hydrated-recipe a `workflow`.

A recipe is a set of instructions with inputs and referenced values to inputs while a workflow is ready-to-be-executed and has no referenced values.

**Comment by theo-armour** [comment]
workflow versus simulation versus ???

With these types of situations, I like to follow the lead of whatever tools
I am working with.

Examples I have dealt with myself

Three.js theta and phi
Mapping: latitude and longitude
Solar calcs: azimuth and bearing

So when dealing with Radiance, copy Radiance terminology. Ditto Dynamo,
Grasshopper.

Do not imagine that peeps will instantly love concepts you just made up one
night. Copy others.

In other words: "Don't make me think", "Don't make me change."

**Comment by AntoineDao** [comment]
Resolved in #61
